tests/basic/kubevirt_test.go

```diff
@@ -827,3 +830,86 @@ var _ = Describe("{KubeVirtPvcAndPoolExpandWithAutopilot}", func() {
 		}
 	})
 })
+
+var _ = Describe("{RebootRootDiskAttachedNode}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootRootDiskAttachedNode", "Reboot the node where VMs root disk is attached", nil, 0)
+		DeployVMTemplatesAndValidate()
+	})
+	var appCtxs []*scheduler.Context
+
+	itLog := "Reboot node where Kubevirt VMs root disk is attached"
+	It(itLog, func() {
+		appList := Inst().AppList
+		defer func() {
+			Inst().AppList = appList
+		}()
+		Inst().AppList = []string{"kubevirt-cirros-live-migration", "kubevirt-windows-vm"}
+		stepLog := "schedule a kubevirtVM"
+		Step(stepLog, func() {
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				log.Infof("RK=> Deploying iteration: %d", i+1)
+				appCtxs = append(appCtxs, ScheduleApplications("reboot")...)
+			}
+		})
+		//defer DestroyApps(appCtxs, nil)
+		ValidateApplications(appCtxs)
+		for _, appCtx := range appCtxs {
+			bindMount, err := IsVMBindMounted(appCtx, false)
+			log.FailOnError(err, "Failed to verify bind mount")
+			dash.VerifyFatal(bindMount, true, "Failed to verify bind mount")
+		}
+
+		stepLog = "Get node where VM's root disk is attached and reboot that node"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+
+			for _, virtualMachineCtx := range appCtxs {
+				bindMount, err := IsVMBindMounted(virtualMachineCtx, false)
+				log.FailOnError(err, "Failed to verify bind mount pre node reboot")
+				dash.VerifyFatal(bindMount, true, "Failed to verify bind mount pre node reboot")
+
+				vms, err := GetAllVMsFromScheduledContexts([]*scheduler.Context{virtualMachineCtx})
+				log.FailOnError(err, "Failed to get VMs from scheduled contexts")
+				dash.VerifyFatal(err, nil, "Failed to to get VMs from scheduled contexts")
```

Comment:
These two lines seem to duplicate the same check. Verify that `len(vms) > 0` here?

Reply:
Fixed.